create-security-certificates.md

```diff
@@ -2,3 +2,68 @@
 title: Create Security Certificates
 toc: false
 ---
+
+A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` command with the appropriate subcommand and flags. 
```

Comment:
Yes, that's what I mean (not all users who manage their own CA will be using openssl, but they'll be comfortable translating those instructions to what they do use, unlike our opaque tool).